templates/zerver/help/read-receipts.md

```diff
@@ -0,0 +1,83 @@
+# Read receipts
+
+Read receipts let you check who has read a message. You can see read receipts
+for any message, including both [stream messages](/help/streams-and-topics) and
+[private messages](/help/private-messages).
+
+With privacy in mind, Zulip lets you [control][configure-personal-read-recipts]
+whether your read receipts are shared, and administrators can
+[choose][configure-organization-read-recipts] whether to enable read receipts in
+their organization.
+
+!!! tip ""
+    Read receipts reflect whether or not someone has marked a message as read,
+    whether by viewing it, or [by marking messages as read in
+    bulk](/help/marking-messages-as-read).
+
+## View who has read a message
+
+{start_tabs}
+
+{!message-actions-menu.md!}
+
+3. Click **View read receipts**.
+
+!!! tip ""
+    You will also see how many people have read the message.
```

Comment:
I think we should add that they can also see the list of names, maybe say something like

```suggestion
    You will also see who has read the message and the total read count.
```

Reply:
Hm, the heading for this section sort of implies the list of names part, but I'll try adding a clarification.